.docker/Dockerfile.cli

```diff
@@ -16,13 +16,22 @@ COPY composer.json composer.lock /app/
 COPY scripts /app/scripts
 
 # Run composer. Additional `rm`s can be added to reduce the image size, with diminishing returns.
-RUN composer install --no-dev \
+RUN composer install --no-dev --quiet \
```

Comment:
please do not use `quiet` - this will silence everything. Instead, use `--no-interaction --no-suggest`. We still want to see which packages at which versions are installed.

Reply:
It was easier to read when i was debugging to be honest. I'm not sure the output is that useful seeing about 500 lines for something that rarely goes wrong - doesn't hide errors?

Reply:
it is not about things going wrong but rather about having a verbose output to know which version is installed in which build.

Consider 2 builds, one of which updates `composer.lock` and fails - it would be useful to know which versions installed when debugging this fail

Reply:
I'm in the process of removing --quiet in a bit of a clean up of progress.